### PR TITLE
feat: enable possibility to customize colors

### PR DESCRIPTION
**Pull request by toshiossada**
Enable to customize the colors of Inspector

usage
```dart
Alice(
  customColors: AliceCustomColors(
    red: Colors.red[900],
    green: Colors.green[800],
  ),
)
```

### CODE REVIEW

File: lib/alice.dart [hunk 1]
```diff
@@ -15,28 +18,20 @@ class Alice {
...
-  );
+  Alice._(this.darkTheme, this._navigatorKey, this._aliceCore,
+      this._httpClientAdapter);
 
   /// Creates alice instance.
   factory Alice({
     GlobalKey<NavigatorState> navigatorKey =
         const GlobalObjectKey<NavigatorState>('AliceNavigatorState'),
     bool darkTheme = false,
+    AliceCustomColors customColors = const AliceCustomColors(),
```
Comment (w3ggy) [inline lib/alice.dart:38]: let's add a comment here with small description and usage example as well.

**Comment by w3ggy** [comment]
Hi! Thank you for your PR.

Please, take a loot at the issues in actions regarding to formatting.

```
Run dart format -o none . --set-exit-if-changed
Changed lib/ui/page/alice_calls_list_screen.dart
Changed lib/ui/widget/alice_call_list_item_widget.dart
Changed lib/utils/alice_constants.dart
Changed lib/utils/alice_custom_colors.dart
Formatted 2[8](https://github.com/Dash-Kit/alice/actions/runs/8498215938/job/23295109625?pr=22#step:6:9) files (4 changed) in 0.48 seconds.
Error: Process completed with exit code 1.
```

Please, fix them

File: lib/alice.dart (hunk 1 above)
Comment (toshiossada) [reply lib/alice.dart:38]: done

**Comment by w3ggy** [comment]
v3.6.0 is deployed

https://pub.dev/packages/alice_lightweight/changelog#360

**Pull request merged**